src/utils/createMultiplierFromDistance.ts

```diff
@@ -0,0 +1,5 @@
+
+export function createMultiplierFromDistance(aX: number, aY: number, bX: number, bY: number) {
```

Comment:
Also, I think that this should be named `getDistanceBetweenTwoPoints` because that's essentially what this does - returns the distance not the multiplier. It _is_ used to get a multiplier as I saw later, but this function is more generic and I believe the name should also be more generic.